ui/js/signup.js

```diff
@@ -319,6 +319,53 @@ require(['jquery', 'underscore', 'oae.core'], function($, _, oae) {
     };
 
     /**
+     * Initialize the signup institution search template
+     */
+    var renderSignUpInstitutionSearch = function() {
+        if (!oae.data.me.tenant.isGuestTenant) {
+            // Don't render institution search at all if we're not on the guest
+            // tenant
+            return;
+        }
+
+        oae.api.util.template().render($('#signup-institution-template'),
+            null, $('#signup-institution-container'));
+
+        oae.api.util.autoSuggest().setup($('#signup-institution-search-field'), {
+            'url': '/api/search/tenants',
+            'formatList': function(data, $el) {
+                var countryInfo = _.findWhere(iso3166.countries, {'code': data.countryCode});
+                oae.api.util.template().render($('#signup-autosuggest-tenant-template'), {
+                    'tenant': _.extend({}, data, {
+                        'countryName': (countryInfo && countryInfo.name),
+                        'countryFlag': (countryInfo && countryInfo.icon)
+                    })
+                }, $el);
+                return $el;
+            },
+            'selectionAdded': function(el) {
```

Comment:
Rather than immediately redirecting the user to the tenant they've selected, should we first show a big black "Go" button next to the autosuggest to have the user confirm that they've selected the correct tenant to avoid them getting lost when they accidentally click the wrong one?

ping @SamPeck : Any thoughts ?